src/pages/Home/SubmitEntries/SuggestAndSubmit/SubmitButton.tsx

```diff
@@ -4,37 +4,36 @@ import { landscapeStyle } from 'styles/landscapeStyle'
 import { useToggle } from 'react-use'
 import AddEntryModal from '../AddEntryModal'
 
+// previous purple color, swapped to signal its in construction
```

Comment:
remove comments

Reply:
wont do, kept to provide context on future development. can be removed when submit is implemented